etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/sftp/SFTP.java

```diff
@@ -158,7 +161,8 @@ private void transferFile(Session session, String remoteFile, String localFile)
                         }
                         fos.write(buf, 0, foo);
                         filesize -= foo;
-                        if (filesize == 0L) break;
+                        if (filesize == 0L)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)